subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -164,68 +164,70 @@ enum node_rx_type {
 #if defined(CONFIG_BT_OBSERVER)
 	/* Advertisement report from scanning */
 	NODE_RX_TYPE_REPORT = 0x04,
-#endif /* CONFIG_BT_OBSERVER */
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
-#if defined(CONFIG_BT_OBSERVER)
 	NODE_RX_TYPE_EXT_1M_REPORT = 0x05,
 	NODE_RX_TYPE_EXT_2M_REPORT = 0x06,
 	NODE_RX_TYPE_EXT_CODED_REPORT = 0x07,
+	NODE_RX_TYPE_EXT_SCAN_TERMINATE = 0x08,
 #if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
-	NODE_RX_TYPE_SYNC = 0x08,
-	NODE_RX_TYPE_SYNC_LOST = 0x09,
+	NODE_RX_TYPE_SYNC = 0x09,
+	NODE_RX_TYPE_SYNC_LOST = 0x0a,
 #endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
+#endif /* CONFIG_BT_CTLR_ADV_EXT */
 #endif /* CONFIG_BT_OBSERVER */
+
 #if defined(CONFIG_BT_BROADCASTER)
-	NODE_RX_TYPE_EXT_ADV_TERMINATE = 0x0a,
-#endif /* CONFIG_BT_BROADCASTER */
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	NODE_RX_TYPE_EXT_ADV_TERMINATE = 0x0b,
 #endif /* CONFIG_BT_CTLR_ADV_EXT */
+#endif /* CONFIG_BT_BROADCASTER */
 
 #if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
-	NODE_RX_TYPE_SCAN_REQ = 0x0b,
+	NODE_RX_TYPE_SCAN_REQ = 0x0c,
 #endif /* CONFIG_BT_CTLR_SCAN_REQ_NOTIFY */
 
 #if defined(CONFIG_BT_CONN)
-	NODE_RX_TYPE_CONNECTION = 0x0c,
-	NODE_RX_TYPE_TERMINATE = 0x0d,
-	NODE_RX_TYPE_CONN_UPDATE = 0x0e,
-	NODE_RX_TYPE_ENC_REFRESH = 0x0f,
+	NODE_RX_TYPE_CONNECTION = 0x0d,
+	NODE_RX_TYPE_TERMINATE = 0x0e,
+	NODE_RX_TYPE_CONN_UPDATE = 0x0f,
+	NODE_RX_TYPE_ENC_REFRESH = 0x10,
 
 #if defined(CONFIG_BT_CTLR_LE_PING)
-	NODE_RX_TYPE_APTO = 0x10,
+	NODE_RX_TYPE_APTO = 0x11,
 #endif /* CONFIG_BT_CTLR_LE_PING */
 
-	NODE_RX_TYPE_CHAN_SEL_ALGO = 0x11,
+	NODE_RX_TYPE_CHAN_SEL_ALGO = 0x12,
 
 #if defined(CONFIG_BT_CTLR_PHY)
-	NODE_RX_TYPE_PHY_UPDATE = 0x12,
+	NODE_RX_TYPE_PHY_UPDATE = 0x13,
 #endif /* CONFIG_BT_CTLR_PHY */
 
 #if defined(CONFIG_BT_CTLR_CONN_RSSI)
-	NODE_RX_TYPE_RSSI = 0x13,
+	NODE_RX_TYPE_RSSI = 0x14,
 #endif /* CONFIG_BT_CTLR_CONN_RSSI */
 #endif /* CONFIG_BT_CONN */
 
 #if defined(CONFIG_BT_CTLR_PROFILE_ISR)
-	NODE_RX_TYPE_PROFILE = 0x14,
+	NODE_RX_TYPE_PROFILE = 0x15,
 #endif /* CONFIG_BT_CTLR_PROFILE_ISR */
 
 #if defined(CONFIG_BT_CTLR_ADV_INDICATION)
-	NODE_RX_TYPE_ADV_INDICATION = 0x15,
+	NODE_RX_TYPE_ADV_INDICATION = 0x16,
 #endif /* CONFIG_BT_CTLR_ADV_INDICATION */
 
 #if defined(CONFIG_BT_CTLR_SCAN_INDICATION)
-	NODE_RX_TYPE_SCAN_INDICATION = 0x16,
+	NODE_RX_TYPE_SCAN_INDICATION = 0x17,
 #endif /* CONFIG_BT_CTLR_SCAN_INDICATION */
 
 #if defined(CONFIG_BT_HCI_MESH_EXT)
-	NODE_RX_TYPE_MESH_ADV_CPLT = 0x17,
-	NODE_RX_TYPE_MESH_REPORT = 0x18,
+	NODE_RX_TYPE_MESH_ADV_CPLT = 0x18,
+	NODE_RX_TYPE_MESH_REPORT = 0x19,
 #endif /* CONFIG_BT_HCI_MESH_EXT */
 
 /* Following proprietary defines must be at end of enum range */
 #if defined(CONFIG_BT_CTLR_USER_EXT)
-	NODE_RX_TYPE_USER_START = 0x18,
+	NODE_RX_TYPE_USER_START = 0x1a,
```

Comment:
This value is incremented by 2 but only one type was inserted above. But if I understand correctly, this change actually fixes a lurking issue.